content/hi/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Windows containers in Kubernetes
+content_type: concept
+weight: 65
+---
+
+<!-- overview -->
+
+Windows एप्लिकेशन कई संगठनों में चलने वाली सेवाओं और अनुप्रयोगों के एक बड़े हिस्से का गठन करते हैं। [Windows कंटेनर](https://aka.ms/windowscontainers) प्रक्रियाओं और पैकेज निर्भरता को समाहित करने का एक तरीका प्रदान करते हैं, जिससे DevOps प्रथाओं का उपयोग करना और विंडोज अनुप्रयोगों के लिए क्लाउड नेटिव पैटर्न का पालन करना आसान हो जाता है। 
+
+विंडोज-आधारित अनुप्रयोगों और लिनक्स-आधारित अनुप्रयोगों में निवेश वाले संगठनों को अपने कार्यभार को प्रबंधित करने के लिए अलग ऑर्केस्ट्रेटर की तलाश करने की आवश्यकता नहीं है, जिससे ऑपरेटिंग सिस्टम की परवाह किए बिना, उनके परिनियोजन में परिचालन क्षमता में वृद्धि होती है।
+
+<!-- body -->
+
+## कुबेरनेट्स में विंडोज नोड्स
+
+कुबेरनेट्स में विंडोज कंटेनरों के ऑर्केस्ट्रेशन को सक्षम करने के लिए, अपने मौजूदा लिनक्स क्लस्टर में विंडोज नोड्स शामिल करें। कुबेरनेट्स पर {{< glossary_tooltip text="पॉड्स" term_id="pod" >}} में विंडोज कंटेनरों को शेड्यूल करना लिनक्स-आधारित कंटेनरों को शेड्यूल करने के समान है।
```

Comment:
> कुबेरनेट्स में विंडोज कंटेनरों के ऑर्केस्ट्रेशन को सक्षम करने के लिए

I am not sure if the plural of containers can be written as कंटेनरों . I believe ऑर्केस्ट्रेशन has a better word too.

Reply:
> I am not sure if the plural of containers can be written as कंटेनरों .
Couldn't figure out a better alternative.

> I believe ऑर्केस्ट्रेशन has a better word too.
I've used it after confirming in the [slack channel](https://kubernetes.slack.com/archives/CJ14B9BDJ/p1637686690074900). The other suggestion was `आयोजन`, should we go for that?